sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -119,15 +119,12 @@ static sycl::detail::ESIMDEmuPluginOpaqueData *PiESimdDeviceAccess;
 // Single-entry cache for piPlatformsGet call.
 static pi_platform PiPlatformCache;
 // TODO/FIXME : Memory leak. Handle with 'piTearDown'.
-static sycl::detail::SpinLock *PiPlatformCacheMutex =
-    new sycl::detail::SpinLock;
+static std::mutex *PiPlatformCacheLock = new std::mutex;
 
 // Mapping between surface index and CM-managed surface
 static std::unordered_map<unsigned int, _pi_mem *> *PiESimdSurfaceMap =
     new std::unordered_map<unsigned int, _pi_mem *>;
-// TODO/FIXME : Memory leak. Handle with 'piTearDown'.
-static sycl::detail::SpinLock *PiESimdSurfaceMapLock =
-    new sycl::detail::SpinLock;
+static std::mutex *PiESimdSurfaceMapLock = new std::mutex;
```

Comment:
why the TODO was removed?

Reply:
Will recover.

Reply:
Done.

Reply:
@dongkyunahn-intel , a question: why just not to fix those global initialization right away? What's the point not to do it properly by initializing and de-initializing in corresponding plugin API ?

I understand that it can be done in a separate commit, but why just not to fix it right away?